index.d.ts

```diff
@@ -327,6 +327,32 @@ Note: This behaviour can be changed with the `skipNull` option.
 */
 export function extract(url: string): string;
 
+/**
+Parses a query string and only picks the passed keysToPicks.
```

Comment:
I think this could be better written. And we don't need to mention parsing the query string.